docs/schema/aws.md

```diff
@@ -946,6 +947,52 @@ Represents an AWS Elastic Load Balancer.  See [spec for details](https://docs.aw
 	(AWSDNSRecord, DNSRecord)-[DNS_POINTS_TO]->(LoadBalancer)
 	```
 
+## LoadBalancerV2
```

Comment:
Please also add entry for `:Endpoint:ELBV2Listener`.

Reply:
`:Endpoint:ELBV2Listener` has been added and links to the documentation.